packages/lesswrong/components/community/modules/SearchResultsMap.tsx

```diff
@@ -132,7 +133,14 @@ const SearchResultsMap = ({center = defaultCenter, zoom = 2, hits, classes}: {
 // connectHits is probably nothing but a consumer acting as a HoC, like this:
 // const connectHits = (C) => { const hits = useHits(); return (args) => C({...args, hits}); }
 // It consumes the data provided by InstantSearch, which should be a parent of us
-const SearchResultsMapComponent = registerComponent("SearchResultsMap", connectHits(SearchResultsMap), { styles });
+type SearchResultsMapProps = {
+  center?: {lat: number, lng: number},
+  zoom?: number,
+  hits?: Array<Hit<AlgoliaUser>>,
+  className?: string
+}
+const ConnectedSearchResultsMap: React.ComponentClass<SearchResultsMapProps, any> = connectHits(SearchResultsMap)
+const SearchResultsMapComponent = registerComponent("SearchResultsMap", ConnectedSearchResultsMap, { styles });
```

Comment:
This was my attempt to fix type errors when trying to pass in props to `SearchResultsMap`. It did seem to work, but let me know if there is a better way to do this.